neo/test/iotest/test_spike2io.py

```diff
@@ -53,6 +53,33 @@ def test_multi_sampling(self):
             assert seg.analogsignals[c].shape == (114618, 1)
             assert seg.analogsignals[c].sampling_rate == 10000 * pq.Hz
 
+    def test_multi_sampling_no_grouping(self):
+        """
+        Some file can have several sampling_rate.
```

Comment:
Rephrasing this would improve readability:
`Some files can contain multiple sampling rates.
This test contains three signals with different sampling rates.`